libafl/Cargo.toml

```diff
@@ -80,6 +82,12 @@ tui = { version = "0.18", default-features = false, features = ['crossterm'], op
 crossterm = { version = "0.24", optional = true }
 clap = {version = "3.2", features = ["derive", "wrap_help"], optional = true}
 
+prometheus-client = "0.18.0" # for prometheus monitor
```

Comment:
These deps should probably be marked as `optional = true` and pulled in explicitly by the `prometheus_monitor` feature above.

Reply:
Good call, added this in [fdce0a7](https://github.com/AFLplusplus/LibAFL/pull/875/commits/fdce0a7c61d2b9b9032539f42270ac6a8a9a509f)